ext/pgsql/php_pgsql.h

```diff
@@ -184,8 +184,8 @@ static const php_stream_ops php_stream_pgsql_fd_ops = {
 ZEND_BEGIN_MODULE_GLOBALS(pgsql)
 	zend_long num_links,num_persistent;
 	zend_long max_links,max_persistent;
-	zend_long allow_persistent;
-	zend_long auto_reset_persistent;
+	bool allow_persistent;
+	bool auto_reset_persistent;
```

Comment:
You may want to also cleanup in the `pgsql.c` conditions like:
```
if (PGG(auto_reset_persistent) & 1)
if (PGG(auto_reset_persistent) & 2)
```

Reply:
not really I need to put it back as previous type, it was done that way on purpose.

Reply:
It's pity that [documentation]([pgsql.auto_reset_persistent](https://www.php.net/manual/en/pgsql.configuration.php#ini.pgsql.auto-reset-persistent)) doesn't describe all possible values for this setting, but `Detect broken persistent links with pg_pconnect(). Needs a little overhead.`

Reply:
true I was thinking this too, I ll look into the doc when I can.